Makefile

```diff
@@ -440,6 +440,9 @@ $(HOME)/go/bin/swagger:
 
 api/openapi-spec/swagger.json: $(HOME)/go/bin/swagger $(SWAGGER_FILES) dist/MANIFESTS_VERSION hack/swaggify.sh
 	swagger mixin -c 412 $(SWAGGER_FILES) | sed 's/VERSION/$(MANIFESTS_VERSION)/' | ./hack/swaggify.sh > api/openapi-spec/swagger.json
+	# Override ParallelSteps definition to match overridden serialization/deserialization logic.  https://github.com/argoproj/argo/issues/2454
```

Comment:
I think there is a better way to do this, modify `_.primary.swagger.json` like this then run `make codegen`:

```
[acollins8@intuitdepe1a95 argo (⎈ |k3s-default:argo) (master)]$ git diff pkg
diff --git a/pkg/apiclient/_.primary.swagger.json b/pkg/apiclient/_.primary.swagger.json
index bef255cc..e3de7577 100644
--- a/pkg/apiclient/_.primary.swagger.json
+++ b/pkg/apiclient/_.primary.swagger.json
@@ -30,6 +30,12 @@
   "definitions": {
     "io.k8s.apimachinery.pkg.runtime.Object": {
       "title": "This is a hack do deal with this problem: https://github.com/kubernetes/kube-openapi/issues/174"
+    },
+    "github.com.argoproj.argo.pkg.apis.workflow.v1alpha1.ParallelSteps": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.WorkflowStep"
+      }
     }
   }
 }
```